### DISCUSSION TITLE
CasaOS的系统日志可以关闭么？

### DISCUSSION THREAD

**Discussion by romeo1978** [ophub/amlogic-s9xxx-armbian#2239] [Rockchip]
想看下armbian的syslog，用journalctl -f 发现全是些casaos的日志，好像每5秒就产生一堆，不知道哪里可以配置关闭？
![image](https://github.com/ophub/amlogic-s9xxx-armbian/assets/8639556/46795a4a-f1ff-474a-ad53-bc4cecf319ca)

**Comment by ophub** [comment]
你到casaos的系统里设置

**Reply by romeo1978** [reply]
好的，多谢o大

**Comment by The4thWall** [comment]
> 好的，多谢o大
你成功关掉了吗？我发现会每隔15分钟写一次日志，导致硬盘休眠又被唤醒

**Comment by viphanf** [comment]
请问如何关闭，没有找到关闭的选项。
